server/services/oneThousandCalculationsService.ts

```diff
@@ -34,6 +34,7 @@ export default class OneThousandCalculationsService {
     const csvData: OneThousandCalculationsRow[] = []
 
     // This is just temporary code therefore the iterative loop rather than an async approach (was easier to develop and easier to debug)
+    // This temporary code and test aid has become a permanent fixture! TODO refactor to coding standards
```

Comment:
This whole one thousand test code has become a permanent fixture... needs tidying (Tech debt ticket)